deployments/arbitrum/usdc/migrations/1679020486_configurate_and_ens.ts

```diff
@@ -0,0 +1,355 @@
+import { BigNumber, Contract, utils } from 'ethers';
+import { expect } from 'chai';
+import { DeploymentManager } from '../../../../plugins/deployment_manager/DeploymentManager';
+import { migration } from '../../../../plugins/deployment_manager/Migration';
+import { calldata, exp, getConfigurationStruct, proposal } from '../../../../src/deploy';
+
+const ENSName = 'compound-community-licenses.eth';
+const ENSResolverAddress = '0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41';
+const ENSSubdomainLabel = 'v3-additional-grants';
+const ENSSubdomain = `${ENSSubdomainLabel}.${ENSName}`;
+const ENSTextRecordKey = 'v3-official-markets';
+
+// https://github.com/OffchainLabs/arbitrum/blob/master/packages/arb-bridge-eth/contracts/libraries/AddressAliasHelper.sol
+function applyL1ToL2Alias(l1Address: string) {
+  const offset = BigNumber.from('0x1111000000000000000000000000000000001111');
+  return BigNumber.from(l1Address).add(offset).toHexString();
+}
+
+const gatewayInterface = new utils.Interface(
+  [
+    'function counterpartGateway() view external returns (address)',
+    'function finalizeInboundTransfer(address _token, address _from, address _to, uint256 _amount, bytes calldata _data) external payable'
+  ]
+);
+
+async function estimateL2Transaction(
+  { from, to, data }: { to: string, from: string, data: string },
+  l2DeploymentManager: DeploymentManager
+) {
+  // guess what the l1 gas price will be when the proposal is executed
+  const l1GasPrice = (utils.parseUnits('33', 'gwei')).toNumber();
```

Comment:
we have a few different places to add buffer to this calculation.

- we can increase the `l1GasPrice` and the `l2GasPrice
- we can add a non-zero `gasBuffer`
- we can increase the buffer on the `l2GasLimit`

Reply:
we should almost certainly bump the l1GasPrice higher. maybe 100 gwei

Reply:
`min(100, currentGas * 2)`?

Reply:
either way, we're making a guess at what the gas price is going to be when the proposal is executed in a week.

we can definitely do a min, but it might be easiest to just be explicit about what the value was.

Reply:
We set this as 100 to be safe.